packages/devtools_app/lib/src/screens/memory/shared/primitives/instance_set_view.dart

```diff
@@ -0,0 +1,95 @@
+// Copyright 2023 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:intl/intl.dart';
+import 'package:vm_service/vm_service.dart';
+
+import '../../../../shared/analytics/constants.dart';
+import '../../../../shared/common_widgets.dart';
+
+typedef SampleObtainer = InstanceRef Function();
+
+class InstanceSetView extends StatelessWidget {
+  const InstanceSetView({
+    super.key,
+    required this.count,
+    required this.sampleObtainer,
+    required this.showMenu,
+    this.textStyle,
```

Comment:
Supernit: I'm not sure if we have a preference overall, but maybe group all the `required` named parameters together?